blocks/v2-columns/v2-columns.css

```diff
@@ -0,0 +1,64 @@
+.v2-columns {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.v2-columns-image,
+.v2-columns-content {
+  max-width: 343px;
+}
+
+.v2-columns-image {
+  display: block;
+}
+
+.v2-columns-content {
+  padding: 40px 0;
+}
+
+.v2-columns-content .v2-columns-content__eyebrow {
+  font-family: var(--ff-accents-regular);
+  font-size: var(--accent-2-font-size);
+  line-height: var(--accent-2-line-height);
+  margin: 0;
+}
+
+.v2-columns-content .v2-columns-content__headline {
+  font-size: var(--headline-1-font-size);
+  line-height: var(--headline-1-line-height);
+  letter-spacing: var(--headline-1-letter-spacing);
+  margin: 12px 0;
+}
+
+.v2-columns-content .v2-columns-content__body {
+  font-size: var(--body-1-font-size);
+  margin: 0 0 32px;
+}
+
+.v2-columns-content a.button {
+  margin: 0;
+}
+
```

Comment:
We should use a native css nesting in the lines above.

Reply:
I agree, but for some reason we haven't been nesting at all